pycbc/events/coinc_rate.py

```diff
@@ -34,20 +35,18 @@ def multiifo_noise_coinc_rate(rates, slop):
         Dictionary keyed on the ifo combination string
         Value is expected coincidence rate in the combination, units Hz
     """
+
     ifos = numpy.array(sorted(rates.keys()))
-    rates_raw = list(rates[ifo] for ifo in ifos)
     expected_coinc_rates = {}
 
     # Calculate coincidence for all-ifo combination
-    # multiply product of trigger rates by the overlap time
-    allowed_area = multiifo_noise_coincident_area(ifos, slop)
-    rateprod = [numpy.prod(rs) for rs in zip(*rates_raw)]
     ifostring = ' '.join(ifos)
-    expected_coinc_rates[ifostring] = allowed_area * numpy.array(rateprod)
+    expected_coinc_rates[ifostring] = combination_noise_coinc_rate(rates, slop)
 
     # if more than one possible coincidence type exists,
     # calculate coincidence for subsets through recursion
     if len(ifos) > 2:
+        ifos = numpy.array(sorted(rates.keys()))
```

Comment:
any reason why they need to be sorted?

Reply:
This line is not actually needed, as it is just repeating line 39

In that case, it _is_ needed to ensure consistency for the ordering of the `ifostring` dict key. Otherwise I think the ordering is not stable

The conversion to a numpy array however is not needed and is removed